src/composables/multisig.ts

```diff
@@ -0,0 +1,220 @@
+import { computed, ref } from '@vue/composition-api';
+import type { Store } from 'vuex';
+import multisigContract from 'ga-multisig-contract/contracts/SimpleGAMultiSig.aes';
+import { uniqBy } from 'lodash-es';
+import camelcaseKeysDeep from 'camelcase-keys-deep';
+import {
+  MemoryAccount, Universal, Node, Crypto,
+} from '@aeternity/aepp-sdk';
+import { decode } from '@aeternity/aepp-sdk/es/tx/builder/helpers';
+import { IAccount, INetwork, IMultisig } from '../types';
+import {
+  fetchJson,
+  postJson,
+  LOCAL_STORAGE_PREFIX,
+} from '../popup/utils';
+import { useSdk } from './sdk';
+import { createPollingBasedOnMountedComponents } from './composablesHelpers';
+import { i18n } from '../store/plugins/languages';
+
+interface UseMultisigOptions {
+  /**
+   * TODO: Temporary solution to avoid dependency circle
+   */
+  store: Store<any>
+}
+
+const POLLING_INTERVAL = 5000;
+const MULTISIG_TRANSCATION_EXPARATION_HEIGHT = 50;
+
+const LOCAL_STORAGE_MULTISIG_KEY = `${LOCAL_STORAGE_PREFIX}_multisig`;
+
+function storeMultisig(multisig: IMultisig[]) {
+  window.localStorage.setItem(LOCAL_STORAGE_MULTISIG_KEY, JSON.stringify(multisig));
```

Comment:
if we use localStorage in other place, i suggest create 2 utils, setToLocalStorage(key, value) and getFromLocalStorage<TYP>(key)

Reply:
This is a good idea but In my opinion we need to postpone this. I think after delivering the Multisig we need to have a nice utility that will take care of syncting the composable state with the Local Storage.

Something similar to this: https://vueuse.org/core/usestorage/

For now I would leave as it is.